CRM/Core/BAO/Mapping.php

```diff
@@ -760,7 +760,7 @@ public static function addComponentFields(&$fields, $mappingType, $exportMode) {
       }
     }
     if (($mappingType == 'Search Builder') || ($exportMode == CRM_Export_Form_Select::GRANT_EXPORT)) {
-      if (CRM_Core_Permission::check('access CiviGrant')) {
+      if (method_exists('CRM_Grant_BAO_Grant', 'exportableFields') && CRM_Core_Permission::check('access CiviGrant')) {
```

Comment:
I think the problem here is that `method_exists` assumes that the class exists and will crash if it doesn't. So I think change this to 
```suggestion
      if (class_exists('CRM_Grant_BAO_Grant') && CRM_Core_Permission::check('access CiviGrant')) {
```

Reply:
The earlier stack trace suggested it was trying to load a selector of some kind. The test ran ok locally. And in fact doesn't even hit the line in the trace(?!) If I do `php -r "var_dump( method_exists('IceCream', 'eat') );"` I get `bool(false)` so php doesn't seem to care (unfortunately I have no IceCream).

The log is a bit quiet about why the test failed on this run although later it has this which is interesting. Let me try something.

```
not ok 3122 - Failure: CRM_Utils_versionCheckTest::testGetSiteStats
  ---
  message: Failed asserting that two arrays are equal.
  severity: fail
  data:
    got:
    - Activity
    - Campaign
    - Case
    - Contact
    - Contribution
    - ContributionPage
    - ContributionProduct
    - Delivered
    - Discount
    - Event
    - Friend
    - Mailing
    - Membership
    - MembershipBlock
    - Participant
    - Pledge
    - PledgeBlock
    - PriceSetEntity
    - Relationship
    - UFGroup
    - Widget
    expected:
    - Activity
    - Campaign
    - Case
    - Contact
    - Contribution
    - ContributionPage
    - ContributionProduct
    - Delivered
    - Discount
    - Event
    - Friend
    - Grant
    - Mailing
    - Membership
    - MembershipBlock
    - Participant
    - Pledge
    - PledgeBlock
    - PriceSetEntity
    - Relationship
    - UFGroup
    - Widget
```